invenio_app_rdm/config.py

```diff
@@ -435,6 +435,7 @@ def files_rest_permission_factory(obj, action):
     "record_export": "/records/<pid_value>/export/<export_format>",
     "record_file_preview": "/records/<pid_value>/preview/<path:filename>",
     "record_file_download": "/records/<pid_value>/files/<path:filename>",
+    "record_from_pid": "/pids/<pid_type>/<path:pid_value>",
```

Comment:
Question: it's not possible to find a way to make them root-level instead?

```/doi/....```

Reply:
Right now we only need for DOIs, so we could hard-coded the DOI prefix.

Reply:
We can I just thought that having it generic was better... and we can not put a root level variable becuase it will conflict with some `/records/<pid_value>`...

I will hardcode doi then.

Reply:
Done